### PR TITLE
Update adopter use case: Hurricane Engines

### PR DESCRIPTION
**Pull request by RainbowMango**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Update adopters' information according to the latest feedback.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/casestudies/adopters.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ To join this list, please follow [these instructions](https://github.com/karmada
...
+| Ecloud                    | https://ecloud.10086.cn/      | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads                                                                                                                                  | TBD                                                                           |
+| Huawei                    | https://www.huaweicloud.com/  | Built the [Ubiquitous Cloud Native Service](https://support.huaweicloud.com/intl/en-us/productdesc-ucs/ucs_productdesc_0001.html) that run applications across regions and clouds with intelligent traffic distribution  | TBD                                                                           |
+| Hurricane Engines         | https://orang.cloud/          | The [ORANG](https://orang.cloud/) `low-code` platform provides the service for users to build powerful apps quickly with zero lines of code, [ORANG](https://orang.cloud/) use Karmada to deploy apps across multi-cloud | [Karmada in AIML INSTITUTE](ci123.md)                                         |
```
Comment (Poor12) [inline docs/casestudies/adopters.md:20]: A small typo.
[ORANG](https://orang.cloud/) uses Karmada to deploy apps across multi-cloud.

Comment (RainbowMango) [reply docs/casestudies/adopters.md]: done.

**Comment by RainbowMango** [comment]
/assign @pidb 
You can get the preview from :
- English: https://deploy-preview-236--karmada.netlify.app/docs/casestudies/adopters
- Chinese: https://deploy-preview-236--karmada.netlify.app/zh/docs/next/casestudies/adopters/

**Review by rgrupesh — COMMENTED** [review]
/lgtm

**Pull request merged**